### PR TITLE
Add Unimplemented Features to Permissions Management

### PR DESCRIPTION
**Pull request by mstratford**
This PR aims to add all of the missing features to managing permissions and their use in actions, training statuses and officerships in MyRadio.

Features now implemented (which where linked to, but not implemented):

- Deleting action permissions.
- Viewing who permissions are "Assigned To", including Officerships and Training Statuses

New Features:
- Adding new permissions.

Moving to another branch:
- (WIP) Displaying current Training Statuses (Studio Trained etc...)
- (WIP) Adding/editing Training Statuses (assigning permissions to them).

### CODE REVIEW

**Comment by mstratford** [comment]
Will complete the training stuff in another branch, this one is getting rather large.

File: src/Classes/MyRadio/AuthUtils.php [hunk 1]
```diff
@@ -340,6 +427,25 @@ public static function addActionPermission($module, $action, $permission)
...
     }
 
+    /**
+     * Deletes action permissions.
+     *
+     * @param int $actPermissionID     The action permission ID.
+     */
+    public static function removeActionPermission($actPermissionID = -1)
+    {
+        if ($actPermissionID >= 0) {
```
Comment (LordAro) [inline src/Classes/MyRadio/AuthUtils.php]: why bother with the check (and default parameter) ? Seems like it's just asking for exceptions

File: src/Controllers/MyRadio/removeActionPermission.php [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        [
+            'enabled' => false
+        ]
+    )
+)->addField(
+    new MyRadioFormField(
+        'confirm',
+        MyRadioFormField::TYPE_CHECK,
+        [
+            'explanation' => '<strong>I confirm that deleting this action permission is
```
Comment (LordAro) [inline src/Controllers/MyRadio/removeActionPermission.php]: Put the name of the permission in the message?

Comment (mstratford) [reply src/Controllers/MyRadio/removeActionPermission.php]: screen shot 2018-04-01 at 17 53 03
It is already above in the form, I deemed that good enough?

**Pull request merged**